app/models/pipeline_run.rb

```diff
@@ -431,7 +431,13 @@ def taxon_byte_ranges_available?
   end
 
   def align_viz_available?
-    # TODO(tiago): we should not have to access aws
+    if Rails.env == "test"
```

Comment:
not sure if the medicine is better than the disease here

Reply:
Do not do this. You should just mock the S3 response... YOu can use Webmock or the AWS SDK stubs.

Reply:
I'll just remove it. 

The issue is only that existing mocking somehow does not work when a single case is called
```
rspec spec/services/pipeline_report_service_spec.rb:165
# does not work
spec/services/pipeline_report_service_spec.rb
# works
```

The error is 
```
     WebMock::NetConnectNotAllowedError:
       Real HTTP connections are disabled. Unregistered request: GET https://idseq-samples-development.s3.us-west-2.amazonaws.com/samples/43/43/postprocess/1.0/subsample_1000000/align_viz.summary with headers ...

       You can stub this request with the following snippet:
     
       stub_request(:get, "https://idseq-samples-development.s3.us-west-2.amazonaws.com/samples/43/43/postprocess/1.0/subsample_1000000/align_viz.summary").
         with(
```

However the suggested stub did not work for me either. Any insight?

By the way, the pattern of returning early for test env exists a couple other places. 

```
  def taxon_search(query, tax_levels = TaxonCount::NAME_2_LEVEL.keys, filters = {})
    return {} if Rails.env == "test"
...

  def list_outputs(s3_path, display_prefix = 1, delimiter = "/")
    return TEST_RESULT_FOLDER if Rails.env == "test"
```

Reply:
Yeah, it is being mocked somewhere else... which is not great, since it is test pollution. We should eventually debug these cases, because it can have unexpected event on tests.
I prefer stubbing the services directly when possible (vs. using generic webmock) like in https://github.com/chanzuckerberg/idseq-web/blob/master/spec/services/sfn_pipeline_dispatch_service_spec.rb#L19